spec/factories/claims.rb

```diff
@@ -11,7 +11,15 @@
 
     after(:build) do |claim, evaluator|
       claim.eligibility = build(*evaluator.eligibility_factory) unless claim.eligibility
-      claim.academic_year = "2019/2020" unless claim.academic_year_before_type_cast
+
+      claim_academic_year =
+        if claim.policy == EarlyCareerPayments
+          AcademicYear::Type.new.serialize(AcademicYear.new([*Date.today.year..Date.today.year + 4].sample))
```

Comment:
@nomasprime why is 2025 included in this array to sample from? I thought the policy for ECP ran for the Academic Years between 2021 thru 2024

Reply:
Offset error and realised I already wrote that part of the line in a nicer way elsewhere, refactoring.